lib/common/test/log.go

```diff
@@ -0,0 +1,25 @@
+package test
+
+import (
+	"os"
+
+	logging "github.com/inconshreveable/log15"
+)
+
+func LogHandler() logging.Handler {
+	handlers := map[string]func() logging.Handler{
+		"null": func() logging.Handler {
+			return logging.DiscardHandler()
+		},
+		"stdout": func() logging.Handler {
+			return logging.StreamHandler(os.Stdout, logging.TerminalFormat())
```

Comment:
Use `StdoutHandler` instead

Reply:
@Geod24 @outsideris 

```
  var  StdoutHandler = StreamHandler(os.Stdout, LogfmtFormat())
```
https://godoc.org/github.com/inconshreveable/log15#pkg-variables

I think `TerminalFormat` is better than `LogfmtFormat` in console :-D

Reply:
It is actually a `TerminalFormat` with support for color, but it's [initialized at runtime](https://github.com/inconshreveable/log15/blob/d4e174bf71609b5afbd4a559f7477f370450053d/root.go#L17-L20).
I've replaced similar code with `StdoutHandler` in #94